src/KernelIntegrator/FlatConnectivity.cc

```diff
@@ -329,17 +326,12 @@ computeGlobalIndices(const DataBase<Dimension>& dataBase,
   VERIFY(numInternalNodesDB == mNumInternalLocalNodes);
   
   // Get global indices manually
-#ifdef USE_MPI
-  int globalScan = 0;
-  MPI_Scan(&mNumInternalLocalNodes, &globalScan, 1, MPI_INT, MPI_SUM, Communicator::communicator());
+  int globalScan = scan(mNumInternalLocalNodes, SPHERAL_OP_SUM);
```

Comment:
`scan()` looks a whole lot like it could be a standard library function call, and at the very least it's not entirely obvious this could be executed in a distributed manner. We might want to either rename this, maybe `distScan`, or namespace the functions so `distributed::scan` & `distributed::allReduce`.